tests/runner.js

```diff
@@ -57,10 +58,9 @@ function checkOnlyInTests() {
 function runMocha() {
   console.time('Mocha Tests Running Time');
   mocha.run(function(failures) {
-    process.on('exit', function() {
-      console.timeEnd('Mocha Tests Running Time');
-      process.exit(failures);
-    });
+    console.timeEnd('Mocha Tests Running Time');
+    exit.releaseExit();
```

Comment:
it is unclear that this doesn't hide other issues though. But maybe as a quickfix this is fine.